home.html

```diff
@@ -0,0 +1,165 @@
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"
+  />
+
+    <title>Capstone project</title>
+</head>
+<!-- social section  -->
+<header class="social">
+    <img src="./images/facebook (1).png" alt="facebook" class="facebook">
+    <img src="./images/twitter (1).png" alt="twitter" class="twitter">
+
+    <ul>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+    </ul>
+</header>
+<body>
+
+    <!-- mobile navigation  -->
+<div class="head">
+    <nav class="navbar">
+        <div class="navv">
+            <div class="ham">
+                <ion-icon name="menu"></ion-icon>
+                <ion-icon name="close-outline" class="close-btn"></ion-icon>
+            </div>
+            <div class="logoo">
+                <div class="logoimg"><a href="./home.html"><img src="./images/thelogo.png" alt="logo" width="100px" height="90px"></a></div>
+                <div class="text">
+                    <p>Click logo!</p>
+                </div>
+            </div>
+        </div>
+        <div class="list_nav">
+            <ul>
+                <li><a href="./home.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#project">Project</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#sponsor">Sponsor</a></li>
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button type="submit" class="campaign">Learn more</button>
+        </div>
+
+        <!-- desktop version  -->
+        <div class="list_navs">
+            <ul>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#project">Project</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#sponsor">Sponsor</a></li>
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button type="submit" class="campaign">Learn more</button>
+        </div>
+        
+    </nav>
+     
+    <!-- hero section  -->
+    <div class="head_content">
+        <h3>"Hello Investors!"</h3>
+        <h2>Find that <br>elusive Unicorns</h2>
+    </div>
+    <div class="header_quote">
+        <p><span><strong>SANO</strong></span> will enable you to discover the newest startups to invest in. It is used by millions of people Worldwide with the best customer satisfaction.</p>
+    </div>
+    <div class="invite_date">
+        <h2><i>COME SEE US IN THE FREE WEBINER</i></h2>
+        <H3>2022:26:11(SAT) ~ 27(SUN)</H3>
+        <p>@International startups Meet, Nigeria, Abuja.</p>
+    </div>
+</div>
+
+<!-- program section  -->
+<div class="programs" id="program">
+    <div class="program">
+        <h3>MAIN PROGRAM</h3>
+        <img src="./images/line2.png" alt="line" width="50px">
+    </div>
+    <div class="programmer">
+        <div class="program_list">
```

Comment:
- [ ] Kindly make sure that the main program columns are aligned properly so your design looks semantic

|Your page|CC design|
|-|-|
|![Screenshot_20221123_202120](https://user-images.githubusercontent.com/25227022/203630217-9cf7ae30-694f-45d7-9fe1-0ee8273a6226.png)|![Screenshot_20221123_202150](https://user-images.githubusercontent.com/25227022/203630232-286d5f36-662f-46be-9d64-ff662dea9e70.png)|